LibGAP.jl/src/gap.jl

```diff
@@ -3,7 +3,8 @@ import Base: convert, getindex, setindex!, length, show
 const True = GAP.GAPFuncs.ReturnTrue()
 const False = GAP.GAPFuncs.ReturnFalse()
 
-const GAPInputType = Union{MPtr,Int64,GAP.GapFFE}
+const GAPInputType_noint = Union{MPtr,GAP.GapFFE}
+const GAPInputType = Union{GAPInputType_noint,Int64}
```

Comment:
Warum eigentlich `GAPInputType`? 

Vielleicht sollte dies der neue `GapObj`-Typ werden?

Reply:
Indeed. We could use `GAPObj` as a name

Reply:
But, I will introduce this in a second PR, as it will need several changes